src/ui/src/containers/live/vega-spec-converters/convert-to-vega-histogram.ts

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
Maybe a little redundant to have these files called `convert-to-vega-*` when they're already nested inside the `vega-spec-converters` directory.
How about just `vega-spec-converters/histogram.ts`?

Reply:
Done.